src/components/TravelSection.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react';
+
+import travelItem01 from '../assets/travel-item-01.png';
+import travelItem02 from '../assets/travel-item-02.png';
+import travelItem03 from '../assets/travel-item-03.png';
+import chevronLeft from '../assets/chevron-left.svg';
+import chevronRight from '../assets/chevron-right.svg';
+
+import styles from './TravelSection.module.css';
+
+interface TravelOption {
+  departure: string;
+  destination: string;
+  type: string;
+  price: number;
+  image: string;
+  link: string;
+}
+
+const travelOptions: TravelOption[] = [
+  {
+    departure: '서울/인천',
+    destination: '두바이',
+    type: '일반석 왕복',
+    price: 1121600,
+    image: travelItem01,
+    link: 'https://koreanairkp.kaltour.com/ProductOverseas/OverseasList?TOURTYP=KALPAK&PKGBRA=KP&PKGARE=E5&REGNB1=%uC720%uB7FD&REGNB2=%uC911%uB3D9&REGTOP=1'
+  },
+  {
+    departure: '서울/인천',
+    destination: '바르셀로나',
+    type: '일반석 왕복',
+    price: 1515200,
+    image: travelItem02,
+    link: 'https://koreanairkp.kaltour.com/ProductOverseas/OverseasView?pkgpnh=KP44129'
+  },
+  {
+    departure: '서울/인천',
+    destination: '로마',
+    type: '일반석 왕복',
+    price: 1415800,
+    image: travelItem03,
+    link: 'https://koreanairkp.kaltour.com/ProductOverseas/OverseasView?pkgpnh=KP41216'
+  }
+];
+
+const TravelSection = () => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const nextTravel = () => {
+    setCurrentIndex((prevIndex) => (prevIndex + 1) % travelOptions.length);
+  };
+
+  const prevTravel = () => {
+    setCurrentIndex((prevIndex) => (prevIndex - 1 + travelOptions.length) % travelOptions.length);
+  };
+
+  const handleCardClick = (link: string) => {
+    window.open(link, '_blank', 'noopener,noreferrer');
+  };
+
+  return (
+    <section className={styles.travelSection}>
+      <button
+        className={`${styles.navButton} ${styles.navButtonPrev}`}
+        onClick={prevTravel}
+        aria-label="이전 여행지 상품으로 이동하기"
+      >
+        <img src={chevronLeft} className={styles.navButtonIcon} alt="" />
+      </button>
+      <article className={styles.carousel} aria-live="polite" aria-atomic="true">
+        {travelOptions.map((option, index) => (
+          <div
+            key={index}
+            className={`${styles.card} ${index === currentIndex ? styles.cardActive : ''}`}
+            onClick={() => handleCardClick(option.link)}
+            role="button"
```

Comment:
오오 치코 꼼꼼한 리뷰 감사해요 :)

[mdn 문서](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role)를 확인해보니 `div` 태그에 `role="button"`을 설정하면 보이스오버가 이를 ‘버튼’로 인식할 수는 있지만, Tab 이동과 키보드 이벤트 처리가 되지 않아서 추가적으로 처리해줘야 하더군요🤣
1. Tab 이동: `div` 태그는 기본적으로 tab을 통해 포커스될 수 없다. 이를 해결하기 위해 `tabindex="0"` 속성을 추가해야 한다. 이렇게 하면 키보드로 해당 요소에 포커스를 이동시킬 수 있다.
    > **Warning:** Be careful when marking up links with the button role. Buttons are expected to be triggered using the Space or Enter key, while links are expected to be triggered using the Enter key. In other words, when links are used to behave like buttons, adding `role="button"` alone is not sufficient. It will also be necessary to add a key event handler that listens for the Space key in order to be consistent with native buttons.

2. 키보드 이벤트 처리: 일반적으로 버튼을 사용할 때는 클릭 이벤트뿐만 아니라 키보드 이벤트(Enter, Space)를 처리해야 한다. `div` 태그에 `role="button"`을 설정한 후, 클릭 외에도 키보드 이벤트 핸들링을 추가해야 한다.
    > Note: If using role="button" instead of the semantic or elements, you will need to make the element focusable and define event handlers for click and keydown events. This includes handling the Enter and Space keypresses in order to process all forms of user input. See the official WAI-ARIA example code.
 
그래서 `tabindex="0"`과 키보드 이벤트 처리를 할지, 아니면 `div` 태그를 `button` 태그로 바꿀지를 고민습니다. 
최종적으로는 두 가지 방법 모두 적용하지 않고, `div` 태그를 `a` 태그로 변경하였어요. 왜냐하면 캐로셀을 클릭할 때 외부 링크로 이동하는 동작이므로 의미론적으로 `a` 태그가 더 적합하다고 생각하였기 때문입니다! 
다만, `a` 태그의 `target="_blank"` 속성을 사용할 때는 보안과 개인정보에 영향을 줄 수 있기 때문에, `rel="noreferrer"`와 `rel="noopener"` 속성을 추가하여 보호했습니다. ([a 태그 mdn 문서](https://developer.mozilla.org/ko/docs/Web/HTML/Element/a#보안과_개인정보))


관련 커밋: 5584600fb9f7bea90f57bc2a44fe721492396cdb